monkey/monkey_island/cc/next_ui/src/providers/theme/ThemeRegistry.tsx

```diff
@@ -1,44 +1,18 @@
 'use client';
 import * as React from 'react';
-import { createTheme, ThemeProvider } from '@mui/material/styles';
-import CssBaseline from '@mui/material/CssBaseline';
-import NextAppDirEmotionCacheProvider from './EmotionCache';
```

Comment:
This must serve some purpose. There are no other uses of it, correct? Is the purpose the same as [this cache from MUI](https://mui.com/material-ui/integrations/nextjs/#configuration)?

Or is the idea that we've switched to styled components and [therefore don't need to configure CSS injection order](https://mui.com/material-ui/integrations/interoperability/#change-the-default-styled-engine)? Though that wouldn't be 100% correct, because we still have styles for the layouts?

@ordabach

Reply:
It will get removed in the next PR